package.json

```diff
@@ -12,7 +12,8 @@
     "es"
   ],
   "scripts": {
-    "test": "cross-env NODE_ENV=test jest --config ./config/jest.config.js --json --outputFile=.jest-test-results.json",
+    "test": "cross-env NODE_ENV=test jest --config ./config/jest.config.js --collectCoverage --json --outputFile=.jest-test-results.json",
+    "test-lintstaged": "cross-env NODE_ENV=test jest --config ./config/jest.config.js --json --outputFile=.jest-test-results.json",
```

Comment:
test-lintstaged --> test:staged

Reply:
--json --outputFile=.jest-test-results.json can also be removed from test-lintstaged

Reply:
Done.